src/brush.js

```diff
@@ -407,7 +408,7 @@ function brush(dim) {
     }
 
     redraw.call(that);
-    emit.start();
+    emit.start(undefined, mode.name);
```

Comment:
```suggestion
    emit.start(event, mode.name);
```

I was really surprised by the undefined event here. Turned out this was a bug in my branch!